src/Microsoft.AspNet.Razor/Chunks/ChunkTreeBuilder.cs

```diff
@@ -27,13 +30,13 @@ public void AddChunk(Chunk chunk, SyntaxTreeNode association, bool topLevel = fa
             chunk.Association = association;
 
             // If we're not in the middle of a parent chunk
-            if (_parentStack.Count == 0 || topLevel == true)
+            if (topLevel == true)
```

Comment:
Not sure why I had `topLevel == true`. Can just use this opportunity to simplify to `if (topLevel)` :smile: